tests/e2e/workloads/pvc_snapshot_and_clone/test_small_files_pvc_cloned_status.py

```diff
@@ -0,0 +1,153 @@
+import logging
```

Comment:
this looks to be  a functional test, please move it under pv_services/pvc_snapshot

Reply:
yes. will change